features/steps/static_primary.py

```diff
@@ -0,0 +1,25 @@
+import json
+import patroni.psycopg as pg
+
+from behave import step, then
```

Comment:
_F401 'behave.then' imported but unused_

```suggestion
from behave import step
```

Reply:
Done.